starter-code/iteration-1/main.ts

```diff
@@ -1,11 +1,35 @@
 // Iteration 1
```

Comment:
We could've packed all this in one very nice class since we are practicing TS and ES6: